Problems building test app

Hello,

first of all, a huge thanks for the wonderful work! Swift/WinRT this is _the_ enabler for using Swift to write modern Windows apps.

I find myself having troubles building Swift/WinRT (or rather, the test app), though. The swiftwinrt.exe is built successfully, but the test app build seems to fail.

### Environment

I'm using Swift 6.0.2 (installed as per [these instructions](https://www.swift.org/install/windows/winget/)) on a Windows 11 VM running on an M1 MBP.

### Prerequisites for Swift/WinRT

The [10.0.17763.0 Windows SDK](https://github.com/thebrowsercompany/swift-winrt/blob/main/CMakePresets.json) referenced in the [swift-winrt](https://github.com/thebrowsercompany/swift-winrt) instructions seems to no longer be available
```
> winget install --id Microsoft.WindowsSDK.10.0.17763
No package found matching input criteria.
```
so I installed a newer one
```
> winget install --id Microsoft.WindowsSDK.10.0.18362
```
updated the `CMakePresets.json` accordingly 
```
"architecture": {
    "value": "arm64",
    "strategy": "external"
},
"cacheVariables": {
    "CMAKE_SYSTEM_VERSION": "10.0.18362.0",
```

### Building Swift/WinRT

Setup:
```
>cmake --preset release
Preset CMake variables:

  CMAKE_BUILD_TYPE="Release"
  CMAKE_CXX_COMPILER="cl"
  CMAKE_C_COMPILER="cl"
  CMAKE_INSTALL_PREFIX:PATH="C:/Users/karl/Documents/sandbox/thebrowsercompany/swift-winrt/out/release"
  CMAKE_MODULE_PATH="C:/Users/karl/Documents/sandbox/thebrowsercompany/swift-winrt/cmake"
  CMAKE_SKIP_INSTALL_ALL_DEPENDENCY:BOOL="TRUE"
  CMAKE_SYSTEM_VERSION="10.0.18362.0"

-- [ NuGet ] Added NuGet Source nuget.org@https://api.nuget.org/v3/index.json
-- [ NuGet ] Package "Microsoft.Windows.CppWinRT.2.0.221121.5" is already installed.
-- Configuring done (8.1s)
-- Generating done (0.0s)
-- Build files have been written to: C:/Users/karl/Documents/sandbox/thebrowsercompany/swift-winrt/build/release
```

Build:
```
>cmake --build --preset release
[0/2] Re-checking globbed directories...
[10/42] Generating ../../../generated/test_component/test_componen...t/impl/test_component.1.h, generated/winrt/impl/test_component.2.
Microsoft (R) 32b/64b MIDLRT Compiler Engine Version 10.00.0231
Copyright (c) Microsoft Corporation. All rights reserved.
64 bit MIDLRT Processing C:\Users\karl\Documents\sandbox\thebrowsercompany\swift-winrt\tests\test_component\cpp\test_component.idl
test_component.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Foundation.idl
Windows.Foundation.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\inspectable.idl
inspectable.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\shared\wtypes.idl
wtypes.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\shared\wtypesbase.idl
wtypesbase.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\shared\basetsd.h
basetsd.h
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\shared\guiddef.h
guiddef.h
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\um\unknwn.idl
unknwn.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\hstring.idl
hstring.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\AsyncInfo.idl
AsyncInfo.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\EventToken.idl
EventToken.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\windowscontracts.idl
windowscontracts.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\IVectorChangedEventArgs.idl
IVectorChangedEventArgs.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\um\oaidl.idl
oaidl.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\um\objidl.idl
objidl.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Storage.Streams.idl
Windows.Storage.Streams.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Storage.idl
Windows.Storage.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Storage.FileProperties.idl
Windows.Storage.FileProperties.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Devices.Geolocation.idl
Windows.Devices.Geolocation.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Storage.Provider.idl
Windows.Storage.Provider.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.UI.idl
Windows.UI.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Storage.Search.idl
Windows.Storage.Search.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Data.Text.idl
Windows.Data.Text.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.UI.Text.Core.idl
Windows.UI.Text.Core.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Globalization.idl
Windows.Globalization.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.System.idl
Windows.System.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.ApplicationModel.idl
Windows.ApplicationModel.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.ApplicationModel.Activation.idl
Windows.ApplicationModel.Activation.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.ApplicationModel.Appointments.AppointmentsProvider.idl
Windows.ApplicationModel.Appointments.AppointmentsProvider.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.ApplicationModel.Appointments.idl
Windows.ApplicationModel.Appointments.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.UI.Popups.idl
Windows.UI.Popups.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.ApplicationModel.Background.idl
Windows.ApplicationModel.Background.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.ApplicationModel.Calls.idl
Windows.ApplicationModel.Calls.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.ApplicationModel.Contacts.idl
Windows.ApplicationModel.Contacts.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.UI.ViewManagement.idl
Windows.UI.ViewManagement.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Devices.Enumeration.idl
Windows.Devices.Enumeration.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Security.Credentials.idl
Windows.Security.Credentials.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Security.Cryptography.Core.idl
Windows.Security.Cryptography.Core.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Security.Cryptography.Certificates.idl
Windows.Security.Cryptography.Certificates.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Networking.idl
Windows.Networking.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Networking.Connectivity.idl
Windows.Networking.Connectivity.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Phone.idl
Windows.Phone.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.UI.Core.idl
Windows.UI.Core.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.UI.Composition.idl
Windows.UI.Composition.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Foundation.Numerics.idl
Windows.Foundation.Numerics.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Graphics.idl
Windows.Graphics.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Graphics.DirectX.idl
Windows.Graphics.DirectX.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Graphics.Effects.idl
Windows.Graphics.Effects.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.UI.Input.idl
Windows.UI.Input.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Devices.Haptics.idl
Windows.Devices.Haptics.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Devices.Input.idl
Windows.Devices.Input.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.UI.Input.Inking.idl
Windows.UI.Input.Inking.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.UI.Input.Inking.Core.idl
Windows.UI.Input.Inking.Core.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.UI.WindowManagement.idl
Windows.UI.WindowManagement.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.ApplicationModel.Calls.Background.idl
Windows.ApplicationModel.Calls.Background.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.ApplicationModel.CommunicationBlocking.idl
Windows.ApplicationModel.CommunicationBlocking.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Devices.Bluetooth.idl
Windows.Devices.Bluetooth.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Devices.Bluetooth.GenericAttributeProfile.idl
Windows.Devices.Bluetooth.GenericAttributeProfile.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Devices.Bluetooth.Rfcomm.idl
Windows.Devices.Bluetooth.Rfcomm.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Networking.Sockets.idl
Windows.Networking.Sockets.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Web.idl
Windows.Web.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Devices.Radios.idl
Windows.Devices.Radios.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Devices.Bluetooth.Advertisement.idl
Windows.Devices.Bluetooth.Advertisement.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Devices.Bluetooth.Background.idl
Windows.Devices.Bluetooth.Background.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Devices.Sensors.idl
Windows.Devices.Sensors.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Graphics.Display.idl
Windows.Graphics.Display.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Devices.SmartCards.idl
Windows.Devices.SmartCards.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Devices.Sms.idl
Windows.Devices.Sms.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Media.Capture.idl
Windows.Media.Capture.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Graphics.DirectX.Direct3D11.idl
Windows.Graphics.DirectX.Direct3D11.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Graphics.Imaging.idl
Windows.Graphics.Imaging.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Media.idl
Windows.Media.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.ApplicationModel.AppService.idl
Windows.ApplicationModel.AppService.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.System.RemoteSystems.idl
Windows.System.RemoteSystems.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Media.Capture.Core.idl
Windows.Media.Capture.Core.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Media.Capture.Frames.idl
Windows.Media.Capture.Frames.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Media.Devices.idl
Windows.Media.Devices.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Media.Devices.Core.idl
Windows.Media.Devices.Core.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Media.MediaProperties.idl
Windows.Media.MediaProperties.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Media.Core.idl
Windows.Media.Core.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Media.Effects.idl
Windows.Media.Effects.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Media.Editing.idl
Windows.Media.Editing.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Media.Transcoding.idl
Windows.Media.Transcoding.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Media.Playback.idl
Windows.Media.Playback.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Media.Audio.idl
Windows.Media.Audio.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Media.Render.idl
Windows.Media.Render.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Media.Casting.idl
Windows.Media.Casting.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Media.Protection.idl
Windows.Media.Protection.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Media.FaceAnalysis.idl
Windows.Media.FaceAnalysis.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Media.Streaming.Adaptive.idl
Windows.Media.Streaming.Adaptive.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Web.Http.idl
Windows.Web.Http.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Web.Http.Filters.idl
Windows.Web.Http.Filters.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Web.Http.Headers.idl
Windows.Web.Http.Headers.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Networking.BackgroundTransfer.idl
Windows.Networking.BackgroundTransfer.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.UI.Notifications.idl
Windows.UI.Notifications.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Data.Xml.Dom.idl
Windows.Data.Xml.Dom.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.UI.Notifications.Management.idl
Windows.UI.Notifications.Management.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Perception.Spatial.idl
Windows.Perception.Spatial.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Perception.idl
Windows.Perception.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Security.Authentication.Web.idl
Windows.Security.Authentication.Web.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.ApplicationModel.Contacts.Provider.idl
Windows.ApplicationModel.Contacts.Provider.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.ApplicationModel.Core.idl
Windows.ApplicationModel.Core.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.ApplicationModel.DataTransfer.ShareTarget.idl
Windows.ApplicationModel.DataTransfer.ShareTarget.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.ApplicationModel.DataTransfer.idl
Windows.ApplicationModel.DataTransfer.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Security.EnterpriseData.idl
Windows.Security.EnterpriseData.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.ApplicationModel.Search.idl
Windows.ApplicationModel.Search.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.ApplicationModel.UserDataAccounts.Provider.idl
Windows.ApplicationModel.UserDataAccounts.Provider.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.ApplicationModel.UserDataAccounts.idl
Windows.ApplicationModel.UserDataAccounts.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.ApplicationModel.Email.idl
Windows.ApplicationModel.Email.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.ApplicationModel.UserDataTasks.idl
Windows.ApplicationModel.UserDataTasks.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.ApplicationModel.Wallet.idl
Windows.ApplicationModel.Wallet.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Devices.Printers.Extensions.idl
Windows.Devices.Printers.Extensions.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Media.SpeechRecognition.idl
Windows.Media.SpeechRecognition.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Security.Authentication.Web.Provider.idl
Windows.Security.Authentication.Web.Provider.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Security.Authentication.Web.Core.idl
Windows.Security.Authentication.Web.Core.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Storage.Pickers.Provider.idl
Windows.Storage.Pickers.Provider.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.UI.WebUI.idl
Windows.UI.WebUI.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Graphics.Printing.idl
Windows.Graphics.Printing.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Web.UI.idl
Windows.Web.UI.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Gaming.Input.idl
Windows.Gaming.Input.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Devices.Power.idl
Windows.Devices.Power.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.System.Power.idl
Windows.System.Power.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Gaming.Input.ForceFeedback.idl
Windows.Gaming.Input.ForceFeedback.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.System.Diagnostics.idl
Windows.System.Diagnostics.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.Data.Json.idl
Windows.Data.Json.idl
64 bit MIDLRT Processing C:\Program Files (x86)\Windows Kits\10\\include\10.0.26100.0\\winrt\Windows.UI.Text.idl
Windows.UI.Text.idl
 tool:  C:\Users\karl\Documents\sandbox\thebrowsercompany\swift-winrt\build\release\Microsoft.Windows.CppWinRT.2.0.221121.5\bin\cppwinrt.exe
 ver:   2.0.221121.5
 in:    C:\Users\karl\Documents\sandbox\thebrowsercompany\swift-winrt\build\release\generated\test_component\test_component.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.AI.MachineLearning.MachineLearningContract\2.0.0.0\Windows.AI.MachineLearning.MachineLearningContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.AI.MachineLearning.Preview.MachineLearningPreviewContract\2.0.0.0\Windows.AI.MachineLearning.Preview.MachineLearningPreviewContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.ApplicationModel.Calls.Background.CallsBackgroundContract\2.0.0.0\Windows.ApplicationModel.Calls.Background.CallsBackgroundContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.ApplicationModel.Calls.CallsPhoneContract\5.0.0.0\Windows.ApplicationModel.Calls.CallsPhoneContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.ApplicationModel.Calls.CallsVoipContract\4.0.0.0\Windows.ApplicationModel.Calls.CallsVoipContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.ApplicationModel.CommunicationBlocking.CommunicationBlockingContract\2.0.0.0\Windows.ApplicationModel.CommunicationBlocking.CommunicationBlockingContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.ApplicationModel.SocialInfo.SocialInfoContract\2.0.0.0\Windows.ApplicationModel.SocialInfo.SocialInfoContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.ApplicationModel.StartupTaskContract\3.0.0.0\Windows.ApplicationModel.StartupTaskContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.Devices.Custom.CustomDeviceContract\1.0.0.0\Windows.Devices.Custom.CustomDeviceContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.Devices.DevicesLowLevelContract\3.0.0.0\Windows.Devices.DevicesLowLevelContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.Devices.Printers.PrintersContract\1.0.0.0\Windows.Devices.Printers.PrintersContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.Devices.SmartCards.SmartCardBackgroundTriggerContract\3.0.0.0\Windows.Devices.SmartCards.SmartCardBackgroundTriggerContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.Devices.SmartCards.SmartCardEmulatorContract\6.0.0.0\Windows.Devices.SmartCards.SmartCardEmulatorContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.Foundation.FoundationContract\3.0.0.0\Windows.Foundation.FoundationContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.Foundation.UniversalApiContract\8.0.0.0\Windows.Foundation.UniversalApiContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.Gaming.XboxLive.StorageApiContract\1.0.0.0\Windows.Gaming.XboxLive.StorageApiContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.Graphics.Printing3D.Printing3DContract\4.0.0.0\Windows.Graphics.Printing3D.Printing3DContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.Networking.Connectivity.WwanContract\2.0.0.0\Windows.Networking.Connectivity.WwanContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.Networking.Sockets.ControlChannelTriggerContract\3.0.0.0\Windows.Networking.Sockets.ControlChannelTriggerContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.Services.Maps.GuidanceContract\3.0.0.0\Windows.Services.Maps.GuidanceContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.Services.Maps.LocalSearchContract\4.0.0.0\Windows.Services.Maps.LocalSearchContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.Services.Store.StoreContract\4.0.0.0\Windows.Services.Store.StoreContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.Services.TargetedContent.TargetedContentContract\1.0.0.0\Windows.Services.TargetedContent.TargetedContentContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.System.Profile.ProfileHardwareTokenContract\1.0.0.0\Windows.System.Profile.ProfileHardwareTokenContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.System.Profile.ProfileSharedModeContract\2.0.0.0\Windows.System.Profile.ProfileSharedModeContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.System.Profile.SystemManufacturers.SystemManufacturersContract\3.0.0.0\Windows.System.Profile.SystemManufacturers.SystemManufacturersContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.System.SystemManagementContract\6.0.0.0\Windows.System.SystemManagementContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.UI.ViewManagement.ViewManagementViewScalingContract\1.0.0.0\Windows.UI.ViewManagement.ViewManagementViewScalingContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.UI.Xaml.Core.Direct.XamlDirectContract\2.0.0.0\Windows.UI.Xaml.Core.Direct.XamlDirectContract.winmd
 out:   C:\Users\karl\Documents\sandbox\thebrowsercompany\swift-winrt\build\release\tests\test_component\cpp\generated\
 cout:  C:\Users\karl\Documents\sandbox\thebrowsercompany\swift-winrt\build\release\tests\test_component\cpp\generated\
 time:  231ms
 tool:  C:\Users\karl\Documents\sandbox\thebrowsercompany\swift-winrt\build\release\Microsoft.Windows.CppWinRT.2.0.221121.5\bin\cppwinrt.exe
 ver:   2.0.221121.5
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.AI.MachineLearning.MachineLearningContract\2.0.0.0\Windows.AI.MachineLearning.MachineLearningContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.AI.MachineLearning.Preview.MachineLearningPreviewContract\2.0.0.0\Windows.AI.MachineLearning.Preview.MachineLearningPreviewContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.ApplicationModel.Calls.Background.CallsBackgroundContract\2.0.0.0\Windows.ApplicationModel.Calls.Background.CallsBackgroundContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.ApplicationModel.Calls.CallsPhoneContract\5.0.0.0\Windows.ApplicationModel.Calls.CallsPhoneContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.ApplicationModel.Calls.CallsVoipContract\4.0.0.0\Windows.ApplicationModel.Calls.CallsVoipContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.ApplicationModel.CommunicationBlocking.CommunicationBlockingContract\2.0.0.0\Windows.ApplicationModel.CommunicationBlocking.CommunicationBlockingContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.ApplicationModel.SocialInfo.SocialInfoContract\2.0.0.0\Windows.ApplicationModel.SocialInfo.SocialInfoContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.ApplicationModel.StartupTaskContract\3.0.0.0\Windows.ApplicationModel.StartupTaskContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.Devices.Custom.CustomDeviceContract\1.0.0.0\Windows.Devices.Custom.CustomDeviceContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.Devices.DevicesLowLevelContract\3.0.0.0\Windows.Devices.DevicesLowLevelContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.Devices.Printers.PrintersContract\1.0.0.0\Windows.Devices.Printers.PrintersContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.Devices.SmartCards.SmartCardBackgroundTriggerContract\3.0.0.0\Windows.Devices.SmartCards.SmartCardBackgroundTriggerContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.Devices.SmartCards.SmartCardEmulatorContract\6.0.0.0\Windows.Devices.SmartCards.SmartCardEmulatorContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.Foundation.FoundationContract\3.0.0.0\Windows.Foundation.FoundationContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.Foundation.UniversalApiContract\8.0.0.0\Windows.Foundation.UniversalApiContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.Gaming.XboxLive.StorageApiContract\1.0.0.0\Windows.Gaming.XboxLive.StorageApiContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.Graphics.Printing3D.Printing3DContract\4.0.0.0\Windows.Graphics.Printing3D.Printing3DContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.Networking.Connectivity.WwanContract\2.0.0.0\Windows.Networking.Connectivity.WwanContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.Networking.Sockets.ControlChannelTriggerContract\3.0.0.0\Windows.Networking.Sockets.ControlChannelTriggerContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.Services.Maps.GuidanceContract\3.0.0.0\Windows.Services.Maps.GuidanceContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.Services.Maps.LocalSearchContract\4.0.0.0\Windows.Services.Maps.LocalSearchContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.Services.Store.StoreContract\4.0.0.0\Windows.Services.Store.StoreContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.Services.TargetedContent.TargetedContentContract\1.0.0.0\Windows.Services.TargetedContent.TargetedContentContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.System.Profile.ProfileHardwareTokenContract\1.0.0.0\Windows.System.Profile.ProfileHardwareTokenContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.System.Profile.ProfileSharedModeContract\2.0.0.0\Windows.System.Profile.ProfileSharedModeContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.System.Profile.SystemManufacturers.SystemManufacturersContract\3.0.0.0\Windows.System.Profile.SystemManufacturers.SystemManufacturersContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.System.SystemManagementContract\6.0.0.0\Windows.System.SystemManagementContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.UI.ViewManagement.ViewManagementViewScalingContract\1.0.0.0\Windows.UI.ViewManagement.ViewManagementViewScalingContract.winmd
 ref:   C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.UI.Xaml.Core.Direct.XamlDirectContract\2.0.0.0\Windows.UI.Xaml.Core.Direct.XamlDirectContract.winmd
 out:   C:\Users\karl\Documents\sandbox\thebrowsercompany\swift-winrt\build\release\tests\test_component\cpp\generated\
 time:  5001ms
[39/42] Shutting down Language Server process
...success
[41/42] Running Swift build...
FAILED: tests/CMakeFiles/test_app tests/aarch64-unknown-windows-msvc/Release/test_app.exe C:/Users/karl/Documents/sandbox/thebrowsercompany/swift-winrt/build/release/tests/CMakeFiles/test_app C:/Users/karl/Documents/sandbox/thebrowsercompany/swift-winrt/build/release/tests/aarch64-unknown-windows-msvc/Release/test_app.exe
C:\WINDOWS\system32\cmd.exe /C "cd /D C:\Users\karl\Documents\sandbox\thebrowsercompany\swift-winrt\tests && swift build -j 2 -Xswiftc -disable-round-trip-debug-types -c release --scratch-path C:/Users/karl/Documents/sandbox/thebrowsercompany/swift-winrt/build/release/tests"
[0/1] Planning build
Building for production...
[0/7] Write auxiliary file C:\Users\karl\Documents\sandbox\thebrowsercompany\swift-winrt\build\release\tests\aarch64-unknown-windows-msvc\release\test_component.build\sources
[1/7] Write auxiliary file C:\Users\karl\Documents\sandbox\thebrowsercompany\swift-winrt\build\release\tests\aarch64-unknown-windows-msvc\release\swift-version--5861058FD09763F4.txt
[2/7] Compiling CWinRT shim.c
warning: save unknown driver flag -disable-round-trip-debug-types as additional swift-frontend flag
[4/8] Compiling test_component aggregation.swift
C:\Users\karl\Documents\sandbox\thebrowsercompany\swift-winrt\tests\test_component\Sources\test_component\Windows.System.swift:15:42: error: cannot find type '__x_ABI_CWindows_CSystem_CUserWatcherUpdateKind' in scope
 13 | public typealias UserWatcherStatus = __x_ABI_CWindows_CSystem_CUserWatcherStatus
 14 | /// [Open Microsoft documentation](https://learn.microsoft.com/uwp/api/windows.system.userwatcherupdatekind)
 15 | public typealias UserWatcherUpdateKind = __x_ABI_CWindows_CSystem_CUserWatcherUpdateKind
    |                                          `- error: cannot find type '__x_ABI_CWindows_CSystem_CUserWatcherUpdateKind' in scope
 16 | /// [Open Microsoft documentation](https://learn.microsoft.com/uwp/api/windows.system.user)
 17 | public final class User : WinRTClass {

C:\Users\karl\Documents\sandbox\thebrowsercompany\swift-winrt\tests\test_component\Sources\test_component\Windows.System+ABI.swift:74:20: error: module 'test_component' has no member named '__x_ABI_C__FIAsyncOperation_1_IInspectableWrapper'
 72 |                 }
 73 |             }
 74 |             return test_component.__x_ABI_C__FIAsyncOperation_1_IInspectableWrapper.unwrapFrom(abi: operation)
    |                    `- error: module 'test_component' has no member named '__x_ABI_C__FIAsyncOperation_1_IInspectableWrapper'
 75 |         }
 76 |

C:\Users\karl\Documents\sandbox\thebrowsercompany\swift-winrt\tests\test_component\Sources\test_component\Windows.System+ABI.swift:85:20: error: module 'test_component' has no member named '__x_ABI_C__FIAsyncOperation_1___x_ABI_C__F__CIPropertySetWrapper'
 83 |                 }
 84 |             }
 85 |             return test_component.__x_ABI_C__FIAsyncOperation_1___x_ABI_C__F__CIPropertySetWrapper.unwrapFrom(abi: operation)
    |                    `- error: module 'test_component' has no member named '__x_ABI_C__FIAsyncOperation_1___x_ABI_C__F__CIPropertySetWrapper'
 86 |         }
 87 |

C:\Users\karl\Documents\sandbox\thebrowsercompany\swift-winrt\tests\test_component\Sources\test_component\Windows.System+ABI.swift:94:20: error: module 'test_component' has no member named '__x_ABI_C__FIAsyncOperation_1___x_ABI_CWindows__CStorage__CStreams__CIRandomAccessStreamReferenceWrapper'
 92 |                 }
 93 |             }
 94 |             return test_component.__x_ABI_C__FIAsyncOperation_1___x_ABI_CWindows__CStorage__CStreams__CIRandomAccessStreamReferenceWrapper.unwrapFrom(abi: operation)
    |                    `- error: module 'test_component' has no member named '__x_ABI_C__FIAsyncOperation_1___x_ABI_CWindows__CStorage__CStreams__CIRandomAccessStreamReferenceWrapper'
 95 |         }
 96 |

C:\Users\karl\Documents\sandbox\thebrowsercompany\swift-winrt\tests\test_component\Sources\test_component\Windows.System+ABI.swift:168:37: error: cannot find '__x_ABI_CWindows_CSystem_CIUserChangedEventArgs2' in scope
166 |         public func get_ChangedPropertyKinds() throws -> test_component.AnyIVectorView<test_component.UserWatcherUpdateKind>? {
167 |             let (value) = try ComPtrs.initialize { valueAbi in
168 |                 _ = try perform(as: __x_ABI_CWindows_CSystem_CIUserChangedEventArgs2.self) { pThis in
    |                                     `- error: cannot find '__x_ABI_CWindows_CSystem_CIUserChangedEventArgs2' in scope
169 |                     try CHECKED(pThis.pointee.lpVtbl.pointee.get_ChangedPropertyKinds(pThis, &valueAbi))
170 |                 }

C:\Users\karl\Documents\sandbox\thebrowsercompany\swift-winrt\tests\test_component\Sources\test_component\Windows.System+ABI.swift:167:39: error: generic parameter 'I' could not be inferred
165 |
166 |         public func get_ChangedPropertyKinds() throws -> test_component.AnyIVectorView<test_component.UserWatcherUpdateKind>? {
167 |             let (value) = try ComPtrs.initialize { valueAbi in
    |                                       `- error: generic parameter 'I' could not be inferred
168 |                 _ = try perform(as: __x_ABI_CWindows_CSystem_CIUserChangedEventArgs2.self) { pThis in
169 |                     try CHECKED(pThis.pointee.lpVtbl.pointee.get_ChangedPropertyKinds(pThis, &valueAbi))

C:\Users\karl\Documents\sandbox\thebrowsercompany\swift-winrt\tests\test_component\Sources\test_component\Support\comptr.swift:87:24: note: in call to function 'initialize'
 85 |     }

 86 |

 87 |     public static func initialize<I>(_ body: (inout UnsafeMutablePointer<I>?) throws -> ()) rethrows -> (ComPtr<I>?) {

    |                        `- note: in call to function 'initialize'
 88 |         var ptr: UnsafeMutablePointer<I>?

 89 |         try body(&ptr)


C:\Users\karl\Documents\sandbox\thebrowsercompany\swift-winrt\tests\test_component\Sources\test_component\Windows.System+ABI.swift:168:25: error: generic parameter 'Type' could not be inferred
166 |         public func get_ChangedPropertyKinds() throws -> test_component.AnyIVectorView<test_component.UserWatcherUpdateKind>? {
167 |             let (value) = try ComPtrs.initialize { valueAbi in
168 |                 _ = try perform(as: __x_ABI_CWindows_CSystem_CIUserChangedEventArgs2.self) { pThis in
    |                         `- error: generic parameter 'Type' could not be inferred
169 |                     try CHECKED(pThis.pointee.lpVtbl.pointee.get_ChangedPropertyKinds(pThis, &valueAbi))
170 |                 }

C:\Users\karl\Documents\sandbox\thebrowsercompany\swift-winrt\tests\test_component\Sources\test_component\Support\iunknown+swift.swift:18:15: note: in call to function 'perform(as:_:)'
16 | extension IUnknown {

17 |   @_alwaysEmitIntoClient @inline(__always)

18 |   public func perform<Type, ResultType>(as type: Type.Type,

   |               `- note: in call to function 'perform(as:_:)'
19 |                                         _ body: (UnsafeMutablePointer<Type>) throws -> ResultType)

20 |       throws -> ResultType {


C:\Users\karl\Documents\sandbox\thebrowsercompany\swift-winrt\tests\test_component\Sources\test_component\Windows.System+ABI.swift:195:20: error: module 'test_component' has no member named '__x_ABI_C__FIAsyncOperation_1___x_ABI_C__FIVectorView_1___x_ABI_CWindows__CSystem__CUserWrapper'
193 |                 }
194 |             }
195 |             return test_component.__x_ABI_C__FIAsyncOperation_1___x_ABI_C__FIVectorView_1___x_ABI_CWindows__CSystem__CUserWrapper.unwrapFrom(abi: operation)
    |                    `- error: module 'test_component' has no member named '__x_ABI_C__FIAsyncOperation_1___x_ABI_C__FIVectorView_1___x_ABI_CWindows__CSystem__CUserWrapper'
196 |         }
197 |

C:\Users\karl\Documents\sandbox\thebrowsercompany\swift-winrt\tests\test_component\Sources\test_component\Windows.System+ABI.swift:204:20: error: module 'test_component' has no member named '__x_ABI_C__FIAsyncOperation_1___x_ABI_C__FIVectorView_1___x_ABI_CWindows__CSystem__CUserWrapper'
202 |                 }
203 |             }
204 |             return test_component.__x_ABI_C__FIAsyncOperation_1___x_ABI_C__FIVectorView_1___x_ABI_CWindows__CSystem__CUserWrapper.unwrapFrom(abi: operation)
    |                    `- error: module 'test_component' has no member named '__x_ABI_C__FIAsyncOperation_1___x_ABI_C__FIVectorView_1___x_ABI_CWindows__CSystem__CUserWrapper'
205 |         }
206 |

C:\Users\karl\Documents\sandbox\thebrowsercompany\swift-winrt\tests\test_component\Sources\test_component\Windows.System+ABI.swift:213:20: error: module 'test_component' has no member named '__x_ABI_C__FIAsyncOperation_1___x_ABI_C__FIVectorView_1___x_ABI_CWindows__CSystem__CUserWrapper'
211 |                 }
212 |             }
213 |             return test_component.__x_ABI_C__FIAsyncOperation_1___x_ABI_C__FIVectorView_1___x_ABI_CWindows__CSystem__CUserWrapper.unwrapFrom(abi: operation)
    |                    `- error: module 'test_component' has no member named '__x_ABI_C__FIAsyncOperation_1___x_ABI_C__FIVectorView_1___x_ABI_CWindows__CSystem__CUserWrapper'
214 |         }
215 |

C:\Users\karl\Documents\sandbox\thebrowsercompany\swift-winrt\tests\test_component\Sources\test_component\Windows.System+ABI.swift:233:37: error: cannot find '__x_ABI_CWindows_CSystem_CIUserStatics2' in scope
231 |         public func GetDefault() throws -> test_component.User? {
232 |             let (result) = try ComPtrs.initialize { resultAbi in
233 |                 _ = try perform(as: __x_ABI_CWindows_CSystem_CIUserStatics2.self) { pThis in
    |                                     `- error: cannot find '__x_ABI_CWindows_CSystem_CIUserStatics2' in scope
234 |                     try CHECKED(pThis.pointee.lpVtbl.pointee.GetDefault(pThis, &resultAbi))
235 |                 }

C:\Users\karl\Documents\sandbox\thebrowsercompany\swift-winrt\tests\test_component\Sources\test_component\Windows.System+ABI.swift:232:40: error: generic parameter 'I' could not be inferred
230 |
231 |         public func GetDefault() throws -> test_component.User? {
232 |             let (result) = try ComPtrs.initialize { resultAbi in
    |                                        `- error: generic parameter 'I' could not be inferred
233 |                 _ = try perform(as: __x_ABI_CWindows_CSystem_CIUserStatics2.self) { pThis in
234 |                     try CHECKED(pThis.pointee.lpVtbl.pointee.GetDefault(pThis, &resultAbi))

C:\Users\karl\Documents\sandbox\thebrowsercompany\swift-winrt\tests\test_component\Sources\test_component\Support\comptr.swift:87:24: note: in call to function 'initialize'
 85 |     }

 86 |

 87 |     public static func initialize<I>(_ body: (inout UnsafeMutablePointer<I>?) throws -> ()) rethrows -> (ComPtr<I>?) {

    |                        `- note: in call to function 'initialize'
 88 |         var ptr: UnsafeMutablePointer<I>?

 89 |         try body(&ptr)


C:\Users\karl\Documents\sandbox\thebrowsercompany\swift-winrt\tests\test_component\Sources\test_component\Windows.System+ABI.swift:233:25: error: generic parameter 'Type' could not be inferred
231 |         public func GetDefault() throws -> test_component.User? {
232 |             let (result) = try ComPtrs.initialize { resultAbi in
233 |                 _ = try perform(as: __x_ABI_CWindows_CSystem_CIUserStatics2.self) { pThis in
    |                         `- error: generic parameter 'Type' could not be inferred
234 |                     try CHECKED(pThis.pointee.lpVtbl.pointee.GetDefault(pThis, &resultAbi))
235 |                 }

C:\Users\karl\Documents\sandbox\thebrowsercompany\swift-winrt\tests\test_component\Sources\test_component\Support\iunknown+swift.swift:18:15: note: in call to function 'perform(as:_:)'
16 | extension IUnknown {

17 |   @_alwaysEmitIntoClient @inline(__always)

18 |   public func perform<Type, ResultType>(as type: Type.Type,

   |               `- note: in call to function 'perform(as:_:)'
19 |                                         _ body: (UnsafeMutablePointer<Type>) throws -> ResultType)

20 |       throws -> ResultType {


C:\Users\karl\Documents\sandbox\thebrowsercompany\swift-winrt\tests\test_component\Sources\test_component\Windows.System+ABI.swift:267:34: error: module 'test_component' has no member named '__x_ABI_C__FITypedEventHandler_2___x_ABI_CWindows__CSystem__CUserWatcher___x_ABI_CWindows__CSystem__CUserChangedEventArgsWrapper'
265 |         public func add_Added(_ handler: TypedEventHandler<test_component.UserWatcher?, test_component.UserChangedEventArgs?>?) throws -> EventRegistrationToken {
266 |             var token: EventRegistrationToken = .init()
267 |             let handlerWrapper = test_component.__x_ABI_C__FITypedEventHandler_2___x_ABI_CWindows__CSystem__CUserWatcher___x_ABI_CWindows__CSystem__CUserChangedEventArgsWrapper(handler)
    |                                  `- error: module 'test_component' has no member named '__x_ABI_C__FITypedEventHandler_2___x_ABI_CWindows__CSystem__CUserWatcher___x_ABI_CWindows__CSystem__CUserChangedEventArgsWrapper'
268 |             let _handler = try! handlerWrapper?.toABI { $0 }
269 |             _ = try perform(as: __x_ABI_CWindows_CSystem_CIUserWatcher.self) { pThis in

C:\Users\karl\Documents\sandbox\thebrowsercompany\swift-winrt\tests\test_component\Sources\test_component\Windows.System+ABI.swift:283:34: error: module 'test_component' has no member named '__x_ABI_C__FITypedEventHandler_2___x_ABI_CWindows__CSystem__CUserWatcher___x_ABI_CWindows__CSystem__CUserChangedEventArgsWrapper'
281 |         public func add_Removed(_ handler: TypedEventHandler<test_component.UserWatcher?, test_component.UserChangedEventArgs?>?) throws -> EventRegistrationToken {
282 |             var token: EventRegistrationToken = .init()
283 |             let handlerWrapper = test_component.__x_ABI_C__FITypedEventHandler_2___x_ABI_CWindows__CSystem__CUserWatcher___x_ABI_CWindows__CSystem__CUserChangedEventArgsWrapper(handler)
    |                                  `- error: module 'test_component' has no member named '__x_ABI_C__FITypedEventHandler_2___x_ABI_CWindows__CSystem__CUserWatcher___x_ABI_CWindows__CSystem__CUserChangedEventArgsWrapper'
284 |             let _handler = try! handlerWrapper?.toABI { $0 }
285 |             _ = try perform(as: __x_ABI_CWindows_CSystem_CIUserWatcher.self) { pThis in

C:\Users\karl\Documents\sandbox\thebrowsercompany\swift-winrt\tests\test_component\Sources\test_component\Windows.System+ABI.swift:299:34: error: module 'test_component' has no member named '__x_ABI_C__FITypedEventHandler_2___x_ABI_CWindows__CSystem__CUserWatcher___x_ABI_CWindows__CSystem__CUserChangedEventArgsWrapper'
297 |         public func add_Updated(_ handler: TypedEventHandler<test_component.UserWatcher?, test_component.UserChangedEventArgs?>?) throws -> EventRegistrationToken {
298 |             var token: EventRegistrationToken = .init()
299 |             let handlerWrapper = test_component.__x_ABI_C__FITypedEventHandler_2___x_ABI_CWindows__CSystem__CUserWatcher___x_ABI_CWindows__CSystem__CUserChangedEventArgsWrapper(handler)
    |                                  `- error: module 'test_component' has no member named '__x_ABI_C__FITypedEventHandler_2___x_ABI_CWindows__CSystem__CUserWatcher___x_ABI_CWindows__CSystem__CUserChangedEventArgsWrapper'
300 |             let _handler = try! handlerWrapper?.toABI { $0 }
301 |             _ = try perform(as: __x_ABI_CWindows_CSystem_CIUserWatcher.self) { pThis in

C:\Users\karl\Documents\sandbox\thebrowsercompany\swift-winrt\tests\test_component\Sources\test_component\Windows.System+ABI.swift:315:34: error: module 'test_component' has no member named '__x_ABI_C__FITypedEventHandler_2___x_ABI_CWindows__CSystem__CUserWatcher___x_ABI_CWindows__CSystem__CUserChangedEventArgsWrapper'
313 |         public func add_AuthenticationStatusChanged(_ handler: TypedEventHandler<test_component.UserWatcher?, test_component.UserChangedEventArgs?>?) throws -> EventRegistrationToken {
314 |             var token: EventRegistrationToken = .init()
315 |             let handlerWrapper = test_component.__x_ABI_C__FITypedEventHandler_2___x_ABI_CWindows__CSystem__CUserWatcher___x_ABI_CWindows__CSystem__CUserChangedEventArgsWrapper(handler)
    |                                  `- error: module 'test_component' has no member named '__x_ABI_C__FITypedEventHandler_2___x_ABI_CWindows__CSystem__CUserWatcher___x_ABI_CWindows__CSystem__CUserChangedEventArgsWrapper'
316 |             let _handler = try! handlerWrapper?.toABI { $0 }
317 |             _ = try perform(as: __x_ABI_CWindows_CSystem_CIUserWatcher.self) { pThis in

C:\Users\karl\Documents\sandbox\thebrowsercompany\swift-winrt\tests\test_component\Sources\test_component\Windows.System+ABI.swift:331:34: error: module 'test_component' has no member named '__x_ABI_C__FITypedEventHandler_2___x_ABI_CWindows__CSystem__CUserWatcher___x_ABI_CWindows__CSystem__CUserAuthenticationStatusChangingEventArgsWrapper'
329 |         public func add_AuthenticationStatusChanging(_ handler: TypedEventHandler<test_component.UserWatcher?, test_component.UserAuthenticationStatusChangingEventArgs?>?) throws -> EventRegistrationToken {
330 |             var token: EventRegistrationToken = .init()
331 |             let handlerWrapper = test_component.__x_ABI_C__FITypedEventHandler_2___x_ABI_CWindows__CSystem__CUserWatcher___x_ABI_CWindows__CSystem__CUserAuthenticationStatusChangingEventArgsWrapper(handler)
    |                                  `- error: module 'test_component' has no member named '__x_ABI_C__FITypedEventHandler_2___x_ABI_CWindows__CSystem__CUserWatcher___x_ABI_CWindows__CSystem__CUserAuthenticationStatusChangingEventArgsWrapper'
332 |             let _handler = try! handlerWrapper?.toABI { $0 }
333 |             _ = try perform(as: __x_ABI_CWindows_CSystem_CIUserWatcher.self) { pThis in

C:\Users\karl\Documents\sandbox\thebrowsercompany\swift-winrt\tests\test_component\Sources\test_component\Windows.System+ABI.swift:347:34: error: module 'test_component' has no member named '__x_ABI_C__FITypedEventHandler_2___x_ABI_CWindows__CSystem__CUserWatcher_IInspectableWrapper'
345 |         public func add_EnumerationCompleted(_ handler: TypedEventHandler<test_component.UserWatcher?, Any?>?) throws -> EventRegistrationToken {
346 |             var token: EventRegistrationToken = .init()
347 |             let handlerWrapper = test_component.__x_ABI_C__FITypedEventHandler_2___x_ABI_CWindows__CSystem__CUserWatcher_IInspectableWrapper(handler)
    |                                  `- error: module 'test_component' has no member named '__x_ABI_C__FITypedEventHandler_2___x_ABI_CWindows__CSystem__CUserWatcher_IInspectableWrapper'
348 |             let _handler = try! handlerWrapper?.toABI { $0 }
349 |             _ = try perform(as: __x_ABI_CWindows_CSystem_CIUserWatcher.self) { pThis in

C:\Users\karl\Documents\sandbox\thebrowsercompany\swift-winrt\tests\test_component\Sources\test_component\Windows.System+ABI.swift:363:34: error: module 'test_component' has no member named '__x_ABI_C__FITypedEventHandler_2___x_ABI_CWindows__CSystem__CUserWatcher_IInspectableWrapper'
361 |         public func add_Stopped(_ handler: TypedEventHandler<test_component.UserWatcher?, Any?>?) throws -> EventRegistrationToken {
362 |             var token: EventRegistrationToken = .init()
363 |             let handlerWrapper = test_component.__x_ABI_C__FITypedEventHandler_2___x_ABI_CWindows__CSystem__CUserWatcher_IInspectableWrapper(handler)
    |                                  `- error: module 'test_component' has no member named '__x_ABI_C__FITypedEventHandler_2___x_ABI_CWindows__CSystem__CUserWatcher_IInspectableWrapper'
364 |             let _handler = try! handlerWrapper?.toABI { $0 }
365 |             _ = try perform(as: __x_ABI_CWindows_CSystem_CIUserWatcher.self) { pThis in

C:\Users\karl\Documents\sandbox\thebrowsercompany\swift-winrt\tests\test_component\Sources\test_component\Windows.System.swift:409:9: error: cannot find '__x_ABI_CWindows_CSystem_CUserType_SystemManaged' in scope
407 |     }
408 |     public static var systemManaged : test_component.UserType {
409 |         __x_ABI_CWindows_CSystem_CUserType_SystemManaged
    |         `- error: cannot find '__x_ABI_CWindows_CSystem_CUserType_SystemManaged' in scope
410 |     }
411 | }

C:\Users\karl\Documents\sandbox\thebrowsercompany\swift-winrt\tests\test_component\Sources\test_component\test_component+ABI.swift:533:17: warning: no calls to throwing functions occur within 'try' expression
 531 |             do {
 532 |                 guard let __unwrapped__instance = IBasicWrapper.tryUnwrapFrom(raw: $0) else { return E_INVALIDARG }
 533 |                 try __unwrapped__instance.method()
     |                 `- warning: no calls to throwing functions occur within 'try' expression
 534 |                 return S_OK
 535 |             } catch { return failWith(error: error) }

C:\Users\karl\Documents\sandbox\thebrowsercompany\swift-winrt\tests\test_component\Sources\test_component\test_component+ABI.swift:535:15: warning: 'catch' block is unreachable because no errors are thrown in 'do' block
 533 |                 try __unwrapped__instance.method()
 534 |                 return S_OK
 535 |             } catch { return failWith(error: error) }
     |               `- warning: 'catch' block is unreachable because no errors are thrown in 'do' block
 536 |         }
 537 |     )
ninja: build stopped: subcommand failed.
```

Any help would be greatly appreciated. Thanks again for bringing us all this Swift goodness on Windows!

--- a/README.md
+++ b/README.md
@@ -29,9 +29,10 @@ git submodule init
 git submodule update --recursive
 ```
 
-Install the required version of the Windows SDK if you don't already have it (currently 10.0.17763):
+Install the required version of the Windows SDK if you don't already have it, currently `10.0.17763`. Note that the WinGet package id for that version of the Windows SDK incorrectly uses `10.0.17736` as the version number.
+
 ```
-winget install --id Microsoft.WindowsSDK.10.0.17763
+winget install --id Microsoft.WindowsSDK.10.0.17736
 ```
 
 ### Integrated Build with CMake

